### PR TITLE
Prevent creating multiple stub mappings with the same ID

### PR DESCRIPTION
**Pull request by RafeArnold**
This pull request implements the fix outlined in [this issue](https://github.com/wiremock/wiremock/issues/2734) to prevent the creation of multiple stub mappings with the same ID.

## References

https://github.com/wiremock/wiremock/issues/2734

## Submitter checklist

- [x] Recommended: Join [WireMock Slack](https://slack.wiremock.org/) to get any help in `#help-contributing` or a project-specific channel like `#wiremock-java`
- [x] The PR request is well described and justified, including the body and the references
- [x] The PR title represents the desired changelog entry
- [x] The repository's code style is followed (see the contributing guide)
- [x] Test coverage that demonstrates that the change works as expected
- [ ] For new features, there's necessary documentation in this pull request or in a subsequent PR to [wiremock.org](https://github.com/wiremock/wiremock.org)

### CODE REVIEW

File: src/test/java/com/github/tomakehurst/wiremock/StubbingAcceptanceTest.java [hunk 1]
```diff
@@ -1221,6 +1224,31 @@ void queryParamCanBeMatchedAsNotAbsent() {
     assertThat(testClient.get("/search").statusCode(), is(404));
   }
 
+  @Test
+  void cannotCreateMultipleStubMappingsWithTheSameId() {
```
Comment (leeturner) [inline src/test/java/com/github/tomakehurst/wiremock/StubbingAcceptanceTest.java:1228]: Is it also worth adding a coresponding test to the `AdminApiTest` to show the errors we get in the admin API when we use a duplicate id ?

Comment (RafeArnold) [reply src/test/java/com/github/tomakehurst/wiremock/StubbingAcceptanceTest.java:1228]: Yes, I think that's a good idea. This test already makes its create request via the admin HTTP client, so is basically already making its assertions on the errors returned by the admin API, but if you think a more explicit test would be worth it, I will add it.

**Review by leeturner — APPROVED** [review]
